integration/docker/entrypoint.sh

```diff
@@ -185,29 +185,6 @@ function setup_for_dynamic_non_root {
   fi
 }
 
-#######################################
-# Sets the Alluxio ram folder if alluxio top tier is MEM and ram folder isn't explicitly configured.
-# Globals:
-#   ALLUXIO_JAVA_OPTS
-#   ALLUXIO_HOME
-#   ALLUXIO_WORKER_JAVA_OPTS
-# Arguments:
-#   None
-#######################################
-function set_ram_folder_if_needed {
```

Comment:
Any reasons to remove this function? Is it unused?

Reply:
There's no tiered store in dora anymore, so the property key in this function is no longer there and thus the function is useless. And Dora doesn't check for whether the storage is mem/ssd/hdd - it just uses the path.